local_server/common/config/client_config.py

```diff
@@ -49,8 +49,8 @@ def get_client_config(app_config, data_adaptor):
         "annotations_cell_ontology_terms": None,
         "custom_colors": dataset_config.presentation__custom_colors,
         "diffexp-may-be-slow": False,
-        "about_legal_tos": dataset_config.app__about_legal_tos,
-        "about_legal_privacy": dataset_config.app__about_legal_privacy,
+        "about_legal_tos": None,
+        "about_legal_privacy": None,
```

Comment:
Because the client uses optional chaining when accessing config params I think you can just remove this. But it would be good to check with @tihuan or @colinmegill

Reply:
Thanks for tagging!

Since DP is now rendering ToS and Privacy, I think we can remove anything related to ToS and Privacy pages in `cellxgene`!

Reply:
Sounds good, removed it.

Reply:
Awesome thanks a lot! 🤩